app/service/parsing_svc.py

```diff
@@ -1,14 +1,48 @@
-import plugins.stockpile.parsers.standard as parsers
-import plugins.stockpile.parsers.mimikatz as mimikatz_parser
+import glob
 from base64 import b64decode
+from pydoc import locate
 
 from app.service.base_service import BaseService
 
 
 class ParsingService(BaseService):
 
     def __init__(self):
+        self.parsers = dict()
         self.log = self.add_service('parsing_svc', self)
+        
+    async def load_parser(self, p_id, name, parser):
+        """
+        Load the specified parser plugin
+        :param p_id:  the id of the yaml file that defines the parser
+        :param name:  the unique name of the parser
+        :param parser:  an instance of the parser class 
+        :return: None
+        """        
+        if name in self.parsers:
+            self.log.warning('Duplicate parser name detected:  %s:%s' % (p_id, name))
+        else:
+            self.parsers[name] = parser       
+        
+    async def load_parsers(self, directory):
+        """
+        Load all parser plugins
+        :param directory:
+        :return: None
+        """
+        for filename in glob.iglob('%s/*.yml' % directory, recursive=False):
```

Comment:
what was the reasoning behind not recursing here?

Reply:
Didn't see any reason why someone would want to create a directory structure under the plugin's parser directory.  This seems to be the same behavior for facts and planners, i.e. no recursion.  Although from performing a quick search, this doesn't seem to be consistent as abilities and adversaries do recurse.  I don't feel strongly either way, if you prefer it to recurse, then so be it.  Fixed.